samcli/cli/cli_config_file.py

```diff
@@ -236,8 +234,10 @@ def decorator_customize_config_file(f):
     config_file_param_decls = ("--config-file",)
     config_file_attrs["help"] = (
         "The path and file name of the configuration file containing default parameter values to use. "
-        "Its default value is 'samconfig.toml' in project directory. For more information about configuration files, "
-        "see: "
+        "Its default value is 'samconfig.toml' in project root directory. Project root directory is defined by the "
```

Comment:
Final tweak, I think the help text being in 3rd person makes it sound more uniform, apologies, I didnt note this on my first pass. But I'm approving the PR, feel free to change the help text to be in 3rd person format.

Reply:
Changing it. Thanks